### ISSUE TITLE
Missing margin of error

### ISSUE THREAD

**Issue by hadley** [walkerke/tidycensus#29]
``` r
library(tidycensus)
census_api_key("2fa891ce3906b073c959075001b6099a4edf7110")
#> To install your API key for use in future sessions, run this function with `install = TRUE`.

geo <- get_acs(
  geography = "metropolitan statistical area/micropolitan statistical area",
  variables = "DP03_0021PE",
  summary_var = "B01003_001",
  survey = "acs1",
  endyear = 2016
)
#> The one-year ACS provides data for geographies with populations of 65,000 and greater.
#> Using the ACS Data Profile
subset(geo, summary_moe < 0 )
#> # A tibble: 394 x 7
#>    GEOID NAME                      variable estim…   moe summary… summary…
#>    <chr> <chr>                     <chr>     <dbl> <dbl>    <dbl>    <dbl>
#>  1 10140 "Aberdeen, WA Micro Area" DP03_00…  1.50  1.10     71628  -5.56e⁸
#>  2 10300 "Adrian, MI Micro Area"   DP03_00…  0.100 0.100    98504  -5.56e⁸
#>  3 10420 "Akron, OH Metro Area"    DP03_00…  1.10  0.300   702221  -5.56e⁸
#>  4 10460 "Alamogordo, NM Micro Ar… DP03_00…  0.700 1.00     65410  -5.56e⁸
#>  5 10540 "Albany, OR Metro Area"   DP03_00…  0     0.100   122849  -5.56e⁸
#>  6 10580 "Albany-Schenectady-Troy… DP03_00…  4.00  0.600   881839  -5.56e⁸
#>  7 10700 "Albertville, AL Micro A… DP03_00…  0.800 0.800    95157  -5.56e⁸
#>  8 10780 "Alexandria, LA Metro Ar… DP03_00…  1.30  0.600   154789  -5.56e⁸
#>  9 10900 "Allentown-Bethlehem-Eas… DP03_00…  2.00  0.400   835652  -5.56e⁸
#> 10 11020 "Altoona, PA Metro Area"  DP03_00…  0.800 0.600   124650  -5.56e⁸
#> # ... with 384 more rows
```

Looks like missing values are encoded as `-555555555` - it'd be nice if this was converted to NA automatically.

**Comment by walkerke** [comment]
Yes, that's a good idea.  I'll make that update.

**Comment by walkerke** [comment]
This is incorporated as of https://github.com/walkerke/tidycensus/commit/ba4ac5e53bb55a41ee9ac5834de23ef65d5adca5.

**Comment by ldaly** [comment]
In the example above, these MSAs have "*****" for the MOE, which indicates that the estimate is controlled. The [Census notes](https://www.census.gov/content/dam/Census/newsroom/press-kits/2015/comparing_acs_estimates.pdf) that these MOEs can be converted to zero when doing statistical tests. 

While I appreciate the simplicity of the current MOE layout, I'd like to be able to differentiate between MOEs for controlled estimates that I might recode to zero, and MOEs that are NA due to too few observations (e.g. "***"). Right now I'm manually verifying MOE types, but it would be great to have a more efficient way to check. 

Would it be possible to add an option to flag the different MOE types, or at least flag the MOEs for controlled estimates vs. all others? 

Thanks!

**Comment by walkerke** [comment]
@ldaly looping back around to this - I'm not seeing how the Census is making this distinction in their documentation so I'll get in touch with them about it.  Incidentally this may get solved as a result of solving another issue (#57).
